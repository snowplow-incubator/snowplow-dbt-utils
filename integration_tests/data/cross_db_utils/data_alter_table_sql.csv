model,id
a,1
