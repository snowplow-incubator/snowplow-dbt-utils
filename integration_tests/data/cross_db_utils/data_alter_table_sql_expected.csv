model,id,last_success,description
a,1,,
